examples/fluids/navierstokes.c

```diff
@@ -727,6 +934,8 @@ int main(int argc, char **argv) {
   CeedScalar theta0      = 300.;     // K
   CeedScalar thetaC      = -15.;     // K
   CeedScalar P0          = 1.e5;     // Pa
+  CeedScalar P_left      = 1.5e5;    // Pa
+  CeedScalar rho_left    = 1.2;      // Kg/m^3
```

Comment:
Should we document why these new parameters are needed in the `fluids/index.rst` file with some mathematical formulation of the new boundary conditions?

Reply:
Yes, you are right. I will add them. Thanks.